src/Microsoft.Framework.WebEncoders.Testing/project.json

```diff
@@ -16,7 +16,7 @@
         },
         "dnx451": {
             "frameworkAssemblies": {
-                "System.Runtime": "4.0.0.0"
+                "System.Runtime": "4.0.10.0"
```

Comment:
So the empty string is correct for `dnx451` framework assemblies? Same for `net45` (e,g, just above)?